lib/tic_tac_toe.rb

```diff
@@ -13,6 +13,13 @@ def initialize(user_interface)
   def start
     display_welcome
     display_board
+
+    display_move_instruction
+
+    position = prompt_move
+    @board.make_move('X', position)
```

Comment:
Is board making a move here?

Reply:
Do you think `place_piece` would be a better name?

Reply:
Create Player class that would be initialized with `marker`
This class will also get the user input, and make the move.